src/app/GitUI/CommandsDialogs/BrowseDialog/DashboardControl/FormDashboardCategoryTitle.cs

```diff
@@ -35,6 +36,7 @@ public FormDashboardCategoryTitle(IEnumerable<string> existingCategories, string
         /// <summary>
         /// Gets the new category.
         /// </summary>
+        [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
```

Comment:
For all these cases I suggest adding `[Browsable(false)]` as well to hide the property from the PropertyGrid.

Reply:
So you mean [Browsable(false)] in addition to [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]?
[Browsable(false)] is set on most occurrences already.

Reply:
> So you mean [Browsable(false)] in addition to [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]?

Yes, it'd be a good idea.

> [Browsable(false)] is set on most occurrences already.

There are some, yes.

Reply:
So with this being an issue in .net9: Should all  [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)] be reverted (and add ignore)?
Keep [Browsable(false)]?

Reply:
@RussKie I guess this has to be used, the error cannot be suppressed globally

Reply:
So, `<NoWarn>$(NoWarn);WFO1000</NoWarn>` doesn't work, does it? In this case, please open a new issue in dotnet/winforms.

/cc: @KlausLoeffelmann